transforms/angle-brackets/transform.js

```diff
@@ -338,7 +348,10 @@ function nodeHasPositionalParameters(node) {
 }
 
 function transformNode(node, fileInfo, config) {
-  if (hasValuelessDataParams(node.params)) {
+  if (
+    hasValuelessDataParams(node.params) &&
+    shouldSkipDataTestDataParams(node, config.includeValuelessDataTestAttributes)
```

Comment:
why not just pass `node.params` instead?

Reply:
Thanks, I thought I might've needed other keys other than params.